### PR TITLE
DLTP-792 change chatty WARNING to DEBUG

### PR DESCRIPTION
**Pull request by msuhovec**
#589

### CODE REVIEW

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ Vagrant.configure(2) do |config|
    end
 
   # CentOS provisioning
-  config.vm.provision "shell", inline: "yum list rpmforge-release || yum -y install http://pkgs.repoforge.org/rpmforge-release/rpmforge-release-0.5.3-1.el6.rf.x86_64.rpm"
+  #config.vm.provision "shell", inline: "yum list rpmforge-release || yum -y install http://pkgs.repoforge.org/rpmforge-release/rpmforge-release-0.5.3-1.el6.rf.x86_64.rpm"
```
Comment (dbrower) [inline vagrant/Vagrantfile]: Just delete the line if it isn't needed?

File: app/helpers/curate_helper.rb [hunk 2]
```diff
@@ -25,7 +25,8 @@ def creator_name_from_pid value
     begin
       p = Person.load_instance_from_solr(value.split("/").last)
     rescue => e
-      logger.warn("WARN: Helper method create_name_from_pid raised an error when loading #{value}.  Error was #{e}")
+      # DLTP-793 This message was filling the production logs- use it only in debug (production log_level is info)
```
Comment (dbrower) [inline app/helpers/curate_helper.rb]: I that you added the bug number in the comment.

File: vagrant/Vagrantfile (hunk 1 above)
Comment (jeremyf) [inline vagrant/Vagrantfile]: Agreed. We have version control to get this information back.

File: app/helpers/curate_helper.rb (hunk 2 above)
Comment (msuhovec) [inline app/helpers/curate_helper.rb]: By intent. You can look it up for details

Comment (dbrower) [inline app/helpers/curate_helper.rb]: I meant to type "I like that you added the bug number in the comment". kudos!

**Pull request merged**